api/flask_app/users/routes.py

```diff
@@ -4,16 +4,27 @@
 from flask_app import db
 from flask_app.models import User
 from flask_app.schemas import UserSchema
-from flask_app.users.forms import TestForm
+from flask_jwt_extended import jwt_required, get_jwt_identity, verify_jwt_in_request, decode_token
+from jwt.exceptions import DecodeError
 from sqlalchemy import or_
+from pprint import pprint
 from werkzeug.security import generate_password_hash
-import time
 
 users_bp = Blueprint("users_bp", __name__, url_prefix="/api/v1/user")
 
 
 @users_bp.route("", methods=["GET"])
+@jwt_required()
 def read_users():
+    headers = request.headers
+    access_token = headers.get("Authorization").split(" ")[-1]
+
+    try:
+        decoded_token = decode_token(encoded_token=access_token)
+        pprint(decoded_token)
```

Comment:
ℹ️ `get_jwt_identity()` will return the same value of `decoded_token["sub"]` if we ever just need the user ID to perform a lookup in our own database / FusionAuth's API for authorization purposes.